images/bridge/ui/session.html

```diff
@@ -9,16 +9,34 @@
   <script src="./webrtc/localmedia.js"></script>
   <link rel="stylesheet" href="./ui/session.css" />
 </head>
-<body class="flex flex-row bg-gray-900 text-white antialiased">
+<body>
+  <div id="body" class="flex flex-row bg-gray-900 text-white antialiased"></div>
   <script type="module">
 
-import {ReflectCommands} from "./ui/substrate-r0.js";
+import {ReflectCommands} from './ui/command-panel.js';
 import {Session} from "./ui/session.js";
 import {Topbar} from "./ui/topbar.js";
 import {decodeMessage} from "./ui/bridge.js";
 import {Session as SFU} from "./webrtc/session.js";
 
-window.substrate.r0.commands = new ReflectCommands();
+let toolCall = new ReflectCommands('https://substrate.home.arpa/tool-call/commands');
+async function suggest(input, commands) {
+  if (!input) {
+    return Object.entries(commands).map(([key, def]) => [key, def, {}]);
+  }
+  const r = await toolCall.run('suggest', { input, commands });
+  return r.choices.flatMap((call) => {
+    const def = commands[call.command];
+    if (!def) return [];
+    return [[call.command, def, call.parameters]];
+  });
+}
```

Comment:
the command-panel should probably provide some helpers for adding the tool-call suggestions, but I moved this out for now to give the user more direct control over suggestions